R/g.plot5.R

```diff
@@ -120,10 +121,10 @@ g.plot5 = function(metadatadir=c(),dofirstpage=FALSE, viewingwindow = 1,f0=c(),f
           daynames = as.character(days2)
           days_SLEEP = g.abr.day.names(daynames)
           # extract variables
-          lengthnight = summarysleep_tmp$acc_SptDuration #including wake periods
```

Comment:
The way the code was here is how it should be.
If there is a sleeplog I think it is better to trust the estimate that uses both sleeplog and accelerometer data (acc_SptDuration). If there is no sleeplog then the difference should be minor.

Reply:
Agreed, best to use the sleeplog if available, only then, revert to detected sleep and wake times

Reply:
I think you misunderstood my comment, my request was to revert this line to
`lengthnight = summarysleep_tmp$acc_SptDuration,`
because that is the estimate of sleep duration that has been tested for validity in the published papers.